app/experimenter/kinto/serializers.py

```diff
@@ -0,0 +1,75 @@
+from datetime import datetime
+from rest_framework import serializers
+
+from experimenter.experiments.models import Experiment, ExperimentVariant
+
+
+class ExperimentRapidBranchesSerializer(serializers.ModelSerializer):
+    value = serializers.SerializerMethodField()
+
+    class Meta:
+        model = ExperimentVariant
+        fields = ("slug", "ratio", "value")
+
+    def get_value(self, obj):
+        # placeholder value
+        return {}
+
+
+class ExperimentRapidArgumentSerializer(serializers.ModelSerializer):
+    slug = serializers.ReadOnlyField(source="normandy_slug")
+    userFacingName = serializers.ReadOnlyField(source="name")
+    userFacingDescription = serializers.ReadOnlyField(source="public_description")
+    active = serializers.ReadOnlyField(default=True)
+    isEnrollmentPaused = serializers.ReadOnlyField(default=False)
+    proposedEnrollment = serializers.ReadOnlyField(source="proposed_enrollment")
+    bucketConfig = serializers.SerializerMethodField()
+    startDate = serializers.SerializerMethodField()
+    endDate = serializers.ReadOnlyField(default=None)
+    branches = ExperimentRapidBranchesSerializer(many=True, source="variants")
+    referenceBranch = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Experiment
+        fields = (
+            "slug",
+            "userFacingName",
+            "userFacingDescription",
+            "active",
+            "isEnrollmentPaused",
+            "features",
+            "proposedEnrollment",
+            "bucketConfig",
+            "startDate",
+            "endDate",
+            "branches",
+            "referenceBranch",
+        )
+
+    def get_bucketConfig(self, obj):
+        return {
+            "randomizationUnit": "normandy_id",
+            "namespace": "",
+            "start": 0,
+            "count": 0,
+            "total": 10000,
+        }
+
+    def get_referenceBranch(self, obj):
+        control_branch = obj.variants.get(is_control=True)
+        return control_branch.slug
+
+    def get_startDate(self, obj):
+        # placeholder value
+        return datetime.today().replace(microsecond=0).isoformat()
```

Comment:
had to chop off the microseconds for the test_task to pass.... 😬

Reply:
re looking at it now, it doesn't make sense to manipulate the implementation to get this [test](https://github.com/mozilla/experimenter/pull/2893/files#diff-a8b743a80bbd875e0a85a68cd6518216R18) to pass, since the times were off in milliseconds [here](https://github.com/mozilla/experimenter/pull/2893/files#diff-a8b743a80bbd875e0a85a68cd6518216R35). I didn't use start_date for this value, since there doesn't seem to be a startDate value that is set with the current form?